### DISCUSSION TITLE
Welcome to kdevcxx_with_ai Discussions!

### DISCUSSION THREAD

**Discussion by arturbac** [arturbac/kdevcxx_with_ai#1] [Announcements]
## Welcome!
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by Petross404** [comment]
Hi, I hope you are doing well.

I forked your repo and cloned it last night. I came across with this way of calling `cmake` outside of the build directory with some arguments.

I foolishly tried to `cmake ..` from inside the build directory and off course compilation failed.

Can you elaborate on this? It's genuine curiosity, don't get me wrong, I am trying to understand how the ebuild should handle this.

Cheers,

Petros

**Reply by arturbac** [reply]
You have chosen wrong disscusion there is for packaging created
https://github.com/arturbac/kdevcxx_with_ai/discussions/2
